search/base.py

```diff
@@ -0,0 +1,16 @@
+"""
+Base test classes for search
+"""
+from django.test import TestCase
+
+from search.api import clear_index
+
+
+class ESTestCase(TestCase):
+    """
+    Set up ES index on setup
+    """
+
+    def setUp(self):
+        super(ESTestCase, self).setUp()
+        clear_index()
```

Comment:
we might want to add an extra function to disable the indexing while we perform some actions in a test: I would prefer to avoid the indexing at all in some cases where I do not need it.
But feel free to ignore for now (and maybe add another issue)